### DISCUSSION TITLE
QuickGrid - RowDetail

### DISCUSSION THREAD

**Discussion by maximilianosilva** [dotnet/aspnetcore#53119] [Ideas]
I would like to see quickgrid have the ability to have RowDetail, similar to what is seen in other Grids Like: https://datatables.net/examples/server_side/row_details.html
https://developer.mescius.com/componentone/docs/blazor/online-blazor/row_details.html
ETC.

But taking advantage of the asynchronous loading that Blazor offers.

**Comment by danroth27** [comment]
Moving this to an issue for consideration: https://github.com/dotnet/aspnetcore/issues/54930
